src/backend/distributed/executor/multi_utility.c

```diff
@@ -167,6 +163,8 @@ multi_ProcessUtility(Node *parsetree,
 					 DestReceiver *dest,
 					 char *completionTag)
 {
+	bool schemaNode = (GetLocalGroupId() == 0);
```

Comment:
An IsSchemaNode function would be nice. We're going to have to call that function a lot to prevent certain commands from the worker.